account_invoice_triple_discount/tests/test_invoice_triple_discount.py

```diff
@@ -165,3 +165,28 @@ def test_04_discounts_decimals_tax(self):
         invoice_form.save()
 
         self.assertEqual(invoice.amount_tax, 177.61)
+
+    def test_05_normal_discount_was_zero(self):
+        """
+        Tests the resetting of the normal discount.
+        """
+        invoice_form = Form(
+            self.AccountMove.with_context(
+                default_move_type="out_invoice",
+                default_journal_id=self.sale_journal.id,
+            )
+        )
+        invoice_form.partner_id = self.partner
+
+        with invoice_form.invoice_line_ids.new() as line_form:
+            line_form.name = "Line 1"
+            line_form.quantity = 10
+            line_form.price_unit = 1.0
+            line_form.discount = 0.0
+            line_form.discount2 = 15.0
+            line_form.tax_ids.clear()
+            line_form.tax_ids.add(self.tax)
+        invoice = invoice_form.save()
+
+        self.assertEqual(invoice.invoice_line_ids[0].discount, 0)
```

Comment:
Maybe you want to add a short explanation how these are the expected values. At least, I find it very helpful in tests to read why the values are the expected ones, especially if I don't know much about this module.